docs/features/tcpproxy.rst

```diff
@@ -3,7 +3,7 @@
 TCP Proxy
 =========
 
-WebSockets or other non-HTTP protocols are not supported by mitmproxy yet. However, you can exempt
+Non-HTTP protocols are not supported by mitmproxy yet. However, you can exempt
```

Comment:
Technically websockets is not a HTTP protocol, so you may want to weaken that statement even further? ☺️